packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -56,12 +57,35 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
       }),
       ['radioInnerCircleSize', 'radioVisibility', 'radioFill'],
     ),
+    content: buildProps(
+      () => ({
+        style: {
+          display: 'flex',
```

Comment:
Was just curious if you need to set display: 'flex' for things to work? I think by default display is already set to 'flex'

Reply:
Removing this line doesn't stylistically change anything, so I suppose it is set by default. Removed, thanks!